### ISSUE TITLE
Adding source with group/len not available yet, any reason?

### ISSUE THREAD

**Issue by ma366** [troglobit/smcroute#135]
I mean what makes the project not supporting that

**Comment by troglobit** [comment]
Not sure I understand what you mean, because we do support it?

Both in /etc/smcroute.conf
```
#   mgroup from IFNAME [source ADDRESS] group MCGROUP[/LEN]
#   mroute from IFNAME [source ADDRESS] group MCGROUP[/LEN] to IFNAME [IFNAME ...
```
and from the command line using smcroutectl
```
add IFNAME [SOURCE] GROUP[/LEN] OUTIFNAME [OUTIFNAME ...]
remove IFNAME [SOURCE] GROUP
join IFNAME [SOURCE] GROUP[/LEN]
leave IFNAME [SOURCE] GROUP[/LEN]
```

Are you perhaps using a pre-packaged SMCRoute from a Linux/UNIX distribution?  Please explain a bit more.

**Comment by ma366** [comment]
When I input the command "smcroutectl add eth0 172.31.3.191 225.1.1.1/24 eth2" it shows source with group/len not supported yet

**Comment by troglobit** [comment]
So ... what version of SMCRoute are you using?

**Comment by ma366** [comment]
2.4.4

**Comment by troglobit** [comment]
Ah, now I see why, sorry!  It seems this support has not yet been released, but will be included in the v2.5 release.

If you like to live on the edge, you can try out the master branch.

**Comment by troglobit** [comment]
Fixed as of 48d049e along with general (S/LEN,G), (S,G/LEN), and (S/LEN, G/LEN) support for both IPv4 and IPv6.

Please note:

> Everything with a non-fixed prefix length (32 or 128) will be treated as
> a dynamic route and only be installed when a matching sender appears.
